modules/ding_place2book/ding_place2book.module

```diff
@@ -1079,52 +1099,45 @@ function ding_place2book_render_place2book_ticketsinfo($node) {
 }
 
 /**
- * Ticketsinfo theme function.
+ * Default theme function for 'ticketsinfo'.
  */
 function theme_place2book_ticketsinfo($variables) {
   $url = $variables['url'];
   $type = $variables['type'];
 
   switch ($type) {
     case 'event-over':
-      $output = '<div class="place2book info">' . t('The event has already taken place') . '</div>';
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('The event has already taken place') . '</div>';
       break;
+
     case 'closed-admission':
-      $output = '<div class="place2book info">' . t('The event is closed for admission') . '</div>';
-      break;
-    case 'sale-not-started':
-      $output = '<div class="place2book info">' . t('Ticket sale has not yet started for this event') . '</div>';
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('Not open for ticket sale') . '</div>';
       break;
+
     case 'no-tickets-left':
-      $output = '<div class="place2book info">' . t('Sold out') . '</div>';
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('Sold out') . '</div>';
       break;
+
     case 'order-link':
-      $output = '<div class="place2book order">' . l(t('Book a ticket'), $url) . '</div>';
+      $output = l(t('Book a ticket'), $url, array('attributes' => array('class' => array('ticket-available', 'button'))));
+      break;
+
+    case 'sale-not-started':
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('Ticket sale has not yet started for this event') . '</div>';
```

Comment:
`sale-not-started` sees to be a new ticket type which is also introduced here but is outside the scope of this PR. Is that intentional?